dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -334,8 +334,9 @@ public class ConnectorArguments extends DefaultArguments {
       parser
           .accepts(
               OPT_THREAD_POOL_SIZE,
-              "Set thread pool size (affects connection pool size). Defaults to "
-                  + OPT_THREAD_POOL_SIZE_DEFAULT)
+              "Set thread pool size (affects connection pool size)."
+                  + " If the --assessment flag is enabled, this option is"
+                  + " ignored and thread pool size is set to 1.")
```

Comment:
Would it make sense to still include the default thread count in this message?

Reply:
The default was duplicated before, since it is also included by the library. After the change it is still visible.

Before:
```
--thread-pool-size <Integer>                               Set thread pool size (affects connection pool size).     
                                                             Defaults to 32 (default: 32)                           
```

After:
```
--thread-pool-size <Integer>                               Set thread pool size (affects connection pool size). If  
                                                             the --assessment flag is enabled, this option is       
                                                             ignored and thread pool size is set to 1. (default: 32)
```